### PR TITLE
Make the JSON validate

### PR DESCRIPTION
**Pull request by ubermichael**
Spot check a few person, title, and firm records and adjust the json so it validates.

### CODE REVIEW

File: templates/firm/show.json.twig [hunk 1]
```diff
@@ -1,23 +1,23 @@
 {
-    "@context": "https://schema.org",
+    "@context": "http://schema.org",
+    "@type": "Organization"
```
Comment (saba1995) [inline templates/firm/show.json.twig:3]: I quite did not understand why do we need to take the organization out of the graph (I think this one also works but I am unsure what was  wrong with the previous version)

File: templates/firm/show.json.twig [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-            "address": {
-                "@type": "PostalAddress"
-                {% if firm.city %},"addressLocality": "{{ firm.city }}" {% endif %}
-                {% if firm.streetAddress %}, "streetAddress": "{{ firm.streetAddress }}" {% endif %}
-            }
-        }
-        {% for titleFirmRole in firm.getTitleFirmRoles %}
-        {% if titleFirmRole.getFirmrole.getName == "Publisher" %}
+        {# the filter in this statement messes with the for loop.last variable
+           so we manually count the items in the list and then check for the last below #}
```
Comment (saba1995) [inline templates/firm/show.json.twig:15]: This totally makes sense.

**Comment by joeytakeda** [comment]
Sorry--didn't see the review request here! This all looks fine to me—happy to merge if it's ready to be.

File: templates/person/show.json.twig [hunk 3]
```diff
@@ -1,50 +1,45 @@
...
-                "geo": {
-                    "@type": "GeoCoordinates",
-                    "latitude": "{{person.cityOfDeath.latitude}}",
-                    "longitude": "{{person.cityOfDeath.longitude}}"
-                }
-            }
-            {% endif %}
-        ,"@id": "{{ url('person_show', {'id':person.id}) }}"
+    "@context": "http://schema.org",
+    "@type": "Person"
```
Comment (saba1995) [inline templates/person/show.json.twig:3]: The same question as the one in firm/show.json.twig exists (why are we taking this out of the graph)

**Comment by ubermichael** [comment]
I really should have done this in a branch, since all the maintenance stuff got bunged up in this.

File: templates/firm/show.json.twig (hunk 1 above)
Comment (ubermichael) [reply templates/firm/show.json.twig:3]: I moved the organization out of the graph and into the root so that the JSON validated.

**Pull request merged**